src/docking/set_module.f90

```diff
@@ -303,6 +307,16 @@ subroutine set_directed(env,key,val,nat,at,idMap,xyz)
          call atl%to_list(list)
          directedset%atoms = list
          directedset%n = size(list)
+      case('expo')
+         if (getValue(env,val,ddum).and.set2) directedset%expo(1) = ddum
+         set2 = .false.
+      case('prefac')
+         !Prefactor is given in kcal, but xtb energies are in hartree
+         if (getValue(env,val,ddum).and.set3) directedset%val(1) = ddum / au
+         set3 = .false.
+      case('midpoint')
+         if (getValue(env,val,ddum).and.set4) directedset%expo(2) = ddum
+         set4 = .false.
```

Comment:
Since these instructions aren't documented in [xcontrol.7.adoc](https://github.com/grimme-lab/xtb/blob/main/man/xcontrol.7.adoc), it might be beneficial to add them to the [xtb-docs](https://xtb-docs.readthedocs.io/en/latest/xtb_docking.html)